x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/prebuilt_rule_customization/customization_enabled/import_rules.ts

```diff
@@ -271,7 +271,7 @@ export default ({ getService }: FtrProviderContext): void => {
         expect(importedRule).toMatchObject({
           rule_id: rule.rule_id,
           version: 9999,
-          rule_source: { type: 'external', is_customized: true },
+          rule_source: { type: 'external', is_customized: false },
```

Comment:
nit: It doesn't look like something besides `version` should be in that assertion.